invenio_rdm_records/contrib/codemeta/custom_fields.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please challenge the placement of these files.

We considered the options:

- `invenio_rdm_records/services/custom_fields/codemeta.py`
- `invenio_rdm_records/services/custom_fields.py`
- `invenio_rdm_records/contrib/codemeta/custom_fields.py` <- implemented option

Reply:
It was suggested to go with the contrib, since extracting the fields might be easier with more structural separation